cni/util/const.go

```diff
@@ -4,7 +4,8 @@ type ExecutionMode string
 
 // CNI execution modes
 const (
-	Default   ExecutionMode = "default"
-	Baremetal ExecutionMode = "baremetal"
-	AKSSwift  ExecutionMode = "aksswift"
+	Default       ExecutionMode = "default"
+	Baremetal     ExecutionMode = "baremetal"
+	AKSSwift      ExecutionMode = "aksswift"
+	V4OnlyOverlay ExecutionMode = "v4OnlyOverlay"
```

Comment:
`/util` is definitely not the right place for these, but I am also not convinced that an additional execution made is the right choice. 

@thatmattlong currently we have an IPAM block in the config:
```json
{
   "cniVersion":"0.3.0",
   "name":"azure",
   "plugins":[
      {
         "type":"azure-vnet",
         "mode":"transparent",
         "ipsToRouteViaHost":["..."],
         "ipam":{
            "type":"azure-cns"
         }
      },
      {
         "type":"portmap",
         "capabilities":{
            "portMappings":true
         },
         "snat":true
      }
   ]
}
```
which is unambiguous in describing Swift (azure-vnet, mode: transparent, ipam: cns). should we add the mode _here_, as maybe 
```json
         "ipam":{
            "type":"azure-cns"
            "mode":"[dynamic|static]"
         }
```

Reply:
/cc @jaer-tsun who might know more about this "execution mode" concept

Reply:
Not sure if ipam is the right place for it, since from CNI perspective the IPAM is just CNS in this case. To me the ipam block describes how CNI goes about getting an IP to use for the Pod, and executionMode describes how it programs the datapath for that IP, which in this case is slightly different than Swift (even though IPAM is exactly the same).

I think either way we go is tricky because a whole new execution mode just to skip adding one chain to iptables feels like overkill, but then putting this flag in ipam also doesn't feel right. Probably eventually CNS should just handle these iptables chains and then there is only one CNI mode.

Reply:
i like the idea of moving swift iptable chains to cns . Even for cilium integration we are considering moving to cns